app/controllers/decidim/vocdoni/admin/steps_controller.rb

```diff
@@ -7,7 +7,8 @@ module Admin
       class StepsController < Admin::ApplicationController
         helper Decidim::ApplicationHelper
         helper StepsHelper
-        helper_method :elections, :election, :current_step
+        helper_method :elections, :election, :current_step, :census_needs_update?,
+                      :non_voters_users_with_authorizations, :users_awaiting_census
```

Comment:
I think `non_voters_users_with_authorizations` is not actually a helper_method, as it isn't being used in the view. Should we remove it from here?